debian/dist/Ubuntu/control

```diff
@@ -162,7 +163,8 @@ Description: MariaDB database client binaries
 
 Package: mariadb-server-core-10.1
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, libmariadbclient18 (>= ${binary:Version})
+Depends: ${shlibs:Depends}, ${misc:Depends}, libmariadbclient18 (>= ${binary:Version}),
+    libsystemd-daemon0
```

Comment:
I did look at the debian docs and those indicated that all libraries of included files have their dependencies included automaticly.